src/server/ua_server_worker.c

```diff
@@ -16,6 +16,11 @@
 
 #include "ua_util.h"
 #include "ua_server_internal.h"
+
+#ifdef UA_ENABLE_SUBSCRIPTIONS_EVENTS
+    #include "ua_subscription_events.h"
```

Comment:
amalgamate.py does not remove indented #include lines, so this line breaks the amalgamated build.

[ 65%] Building C object CMakeFiles/open62541-object.dir/open62541.c.o
/home/jvoe/open62541-events/build/open62541.c:186931:40: fatal error: ua_subscription_events.h: No such file or directory
compilation terminated.

If the indentation is removed, the amalgamated build works as expected.